core/build.gradle

```diff
@@ -41,9 +41,9 @@ mrJarVersions.each { version->
         testClassesDirs = sourceSets."java${version}Test".output.classesDirs
         classpath = sourceSets."java${version}Test".runtimeClasspath
 
-        enabled = (Integer.parseInt(JavaVersion.current().getMajorVersion()) >= version) &&
-                (!rootProject.hasProperty('testJavaVersion') ||
-                        Integer.parseInt(rootProject.findProperty('testJavaVersion')) >= version)
+        enabled = Integer.parseInt(JavaVersion.current().getMajorVersion()) >= version &&
+                rootProject.ext.testJavaVersion != null &&
```

Comment:
IIUC, `rootProject.ext.testJavaVersion` is not null always?

Reply:
Yeah, it's always not null. 😄